### ISSUE TITLE
Failing rendering tests: ol.rendering.style.RegularShape

### ISSUE THREAD

**Issue by ahocevar** [openlayers/ol3#3561]
```
tests the canvas renderer ‣
Error: expected '9.40' to be below 6 (http://localhost:3000/node_modules/expect.js/index.js:102)
onerror@http://localhost:3000/node_modules/mocha/mocha.js:5990:19

tests the WebGL renderer ‣
Error: expected '7.16' to be below 1.5 (http://localhost:3000/node_modules/expect.js/index.js:102)
onerror@http://localhost:3000/node_modules/mocha/mocha.js:5990:19
```

**Comment by ahocevar** [comment]
Do these tests fail for anyone else?

**Comment by ahocevar** [comment]
Also, I noticed that the result looks different for the rotated triangle in Canvas and WebGL. Looks like one of the renderers calculates the rotation incorrectly.

**Comment by tsauerwein** [comment]
Depending on the browser and the OS the rendered image might differ. That's why we have to use a tolerance at all.

I think we can increase the tolerance for these two test cases to make them pass.

> Also, I noticed that the result looks different for the rotated triangle in Canvas and WebGL. Looks like one of the renderers calculates the rotation incorrectly.

I created a ticket for this: https://github.com/openlayers/ol3/issues/3563

**Comment by tschaub** [comment]
> I think we can increase the tolerance for these two test cases to make them pass.

Is the rotation issue not part of the failure?  If not, I guess we'll want to add a regression test as part of the fix for #3563.

**Comment by tsauerwein** [comment]
> Is the rotation issue not part of the failure?

Not directly. The are two different reference images, one for WebGL and one for canvas. But yes, we'll have to update either the image for WebGL or canvas once #3563 is fixed.

**Comment by fredj** [comment]
fixed with #6169
